js/Paddle.js

```diff
@@ -4,30 +4,43 @@ class Paddle {
         this.ctx = canvasConfig.ctx; // pobranie wartości ctx z gettera
         this.paddleHeight = 10; // wysokość paletki
         this.paddleWidth = 75; //szerokość paletki
-        this.paddleX = (this.canvas.width-this.paddleWidth)/2; // pozycja paletki na osi X
+        this.paddleX = (canvasConfig.cvs.width-this.paddleWidth)/2; // pozycja paletki na osi X
+        this.canvasWidth = canvasConfig.cvs.width; // szerokość canvas
         this.rightPressed = false; // czy prawa strzałka jest obecnie wciśnięta
         this.leftPressed = false; // czy lewa strzałka jest obecnie wciścnięta
     }
 
     drawPaddle(){
         // TO DO -> funkcja tworząca paletkę
         // hint: korzystajcie z wartości zdefiniowanych w constructorze oraz metod canvas'a
+        ctx.fillRect(this.paddleX,this.cvs.height-this.paddleHeight,this.paddleWidth,this.paddleHeight);
     };
 
     movePaddle(){
-        // TO DO -> funkcja ruszające paletką na podstawie tego co zwracają eventListenery w index.js
+        //funkcja ruszające paletką na podstawie tego co zwracają eventListenery w index.js
         // hint: "szybkość" = 7
+        if(this.rightPressed && this.paddleX<(this.canvasWidth-this.paddleWidth))
+        {
+            this.paddleX+=7;
+        }
+        if(this.leftPressed && this.paddleX>0)
+        {
+            this.paddleX-=7;
+        }
     };
 
-    setRightPressed(){
-        // TO DO -> setter zmieniający wartość this.rightPressed
+    setRightPressed(bool){
+        //setter zmieniający wartość this.rightPressed
+        this.rightPressed = bool;
     };
 
-    setLeftPressed(leftPressed){
-        // TO DO -> setter zmieniający wartość this.leftPressed
+    setLeftPressed(bool){
+        //setter zmieniający wartość this.leftPressed
+        this.leftPressed = bool;
     };
 
     getPaddleConfig(){
         // TO DO -> getter zwracający: this.paddleWidth, this.paddleX, this.rightPressed i this.leftPressed
+        return [this.paddleWidth,this.paddleX,this.rightPressed,this.leftPressed];
```

Comment:
zwrócenie tego w postaci tablicy nie jest błędem, ale potem będzie to wykorzystywane przy obijaniu piłeczki od paletki i prościej będzie jak zwrócisz to jako obiekt ;)